ExeterTeam25/group-software-project
# Weekly Challenge System implementation

WARNING: Make sure to run makemigrations and migrate before you run this!

To test auto generation:
- Run the server and go into main page
- Verify weekly challenges appear in map
- Go to admin page
- Either delete all those generated challenges or change their date so they're expired
- Go to main page and make sure that new challenges have appeared
- Log in as privileged user
- Make sure you can't make challenge with "Weekly" category

Details:
Added Default challenges JSON
Implemented auto generating weekly challenges
Made it so "Weekly " category auto generates
Created icons and proper rendering
